demo/ez-selection-activate-element.html

```diff
@@ -0,0 +1,55 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+        <title>ez-selection-activate-element demo</title>
+
+        <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+
+        <link rel="import" href="../../iron-demo-helpers/demo-snippet.html">
+        <link rel="import" href="../ez-selection-activate-element.html">
+
+    </head>
+    <body>
+        <div class="vertical-section-container centered">
+            <h3>Enable a button based on a checkboxes</h3>
```

Comment:
`Enable a button based on checkboxes` (that was one of the point of the comment on that line in the previous review round)